package.json

```
@@ -1,22 +1,51 @@
 {
 	"name": "resolume-arena",
-	"version": "2.0.2",
-	"main": "src/index.js",
-	"type": "module",
+	"main": "dist/index.js",
+	"version": "3.0.0",
+	"description": "Resolume Arena 6 & 7 module for companion",
+	"api_version": "1.0.0",
+	"keywords": [
+		"Video",
+		"Software",
+		"Graphics"
+	],
+	"manufacturer": "Resolume",
+	"product": "Arena",
+	"shortname": "arena",
+	"homepage": "https://github.com/bitfocus/companion-module-resolume-arena#readme",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1",
-		"format": "prettier --write ."
+		"dev": "yarn build:watch",
+		"build": "npx rimraf dist && npx -p typescript tsc -p tsconfig.build.json",
```

```suggestion
		"build": "rimraf dist && tsc -p tsconfig.build.json",
```
No need to use `npx` for a module in the v3 format.
You might need to add `rimraf` as a dev dependency

---

done